scripts/clean_unused_imports.py

```diff
@@ -2,10 +2,12 @@
 import os
 import re
 
-IMPORT_RE = re.compile(r'^\s*import\s+[\w\.]+\.(\w+)\s*;\s*(?://.*)?$')
+IMPORT_RE_JAVA = re.compile(r'^\s*import\s+[\w\.]+\.(\w+)\s*;\s*(?://.*)?$')
+IMPORT_RE_PHP = re.compile(r'^\s*use\s+[\w\\]+\\(\w+)\s*\w*\s*(\w*);\s*(?://.*)?$')
 
 LANGUAGE_REGEX_MAP = {
-    "java": "*.java"
+    "java": "*.java",
+    "php": "*.php"
 }
```

Comment:
Should turn this into a multi-dimensional map so there's less conditional logic, like:
```
LANGUAGE_REGEX_MAP = {
    "java": {
        "pattern": "*.java",
        "regex": re.compile(r'^\s*import\s+[\w\.]+\.(\w+)\s*;\s*(?://.*)?$'),
    }
    ...
}
```